docgen/src/examples/tourism/App.js

```diff
@@ -30,6 +30,7 @@ const App = props =>
     onSearchStateChange={props.onSearchStateChange.bind(this)}
   >
     <Configure aroundLatLngViaIP={true}/>
+    <Configure aroundRadius="all"/>
```

Comment:
We can reuse the same Configure (otherwise people might read "1 
 Configure = 1 param)